docs/guides/remote.rst

```diff
@@ -33,13 +33,13 @@ When you run training, specify the full path to the :code:`json` file.
 
 **Command-line training**:
 
-Once you have your training package (or labels project file) and training profile, you can run training like so:
+Once you have your training job package (or labels package and training profile), you can run training like so:
 
 ::
 
-  sleap-train path/to/your/training_profile.json another/path/to/training_package.h5
+  sleap-train path/to/your/training_profile.json another/path/to/labels_package.h5
```

Comment:
`labels.pkg.slp`, not `.h5` (this is old)

Reply:
Done